README.md

```diff
@@ -1,10 +1,24 @@
 # Proof of Concept: HPO on FHIR
 
-This simple application can search for patients in the hardcoded sandbox and display LOINC Codes within observations along with the corresponding HPO Term if one exists. You can also see all the LOINC Codes that have been annotated and the raw annotations for each.
+This simple application can search for patients in a hardcoded sandbox and display LOINC Codes within observations along with the corresponding HPO Term if one exists. You can also see all the LOINC Codes that have been annotated and the raw annotations for each.
 
 Running the app will require the following:
 
-- Locally install the phenol library which is a dependency of the fhir2hpo library: https://github.com/monarch-initiative/phenol
-- Locally install the fhir2hpo library: https://github.com/OCTRI/fhir2hpo
+- Install Maven for building code. Run ```mvn clean install` on the following libraries to get them into your local repository. Run these in order:
+- The phenol library: https://github.com/monarch-initiative/phenol
+- The fhir2hpo library: https://github.com/OCTRI/fhir2hpo
+- This application
```

Comment:
Is this bullet missing some text?

Reply:
No, but I can see how this is confusing. I was listing this application as a dependency of running this application. Duh! I'll remove.

Reply:
haha okay I get it now.